test/parse/big-file-performance.test.js

```diff
@@ -1,180 +0,0 @@
-/* eslint strict: off */
-
-var assert = require('assert')
-var XMLSerializer = require('../../lib/dom-parser').XMLSerializer
-var DOMParser = require('../../lib/dom-parser').DOMParser
-var DomJS = require('dom-js').DomJS
-try {
-	var Libxml = require('libxmljs')
-	console.log('libxml loaded!!')
-} catch (e) {
-	console.error(e)
-}
-
-function xmldom(data) {
-	console.time('xmldom')
-	var doc = new DOMParser({ locator: null, checkLater: true }).parseFromString(
-		data
-	)
-	console.timeEnd('xmldom')
-	if (doc.toString.name != 'xxxxx') {
-		doc.toString = function xxxxx() {
-			return new XMLSerializer().serializeToString(doc)
-		}
-	}
-	return doc
-}
-function libxml(data) {
-	if (Libxml) {
-		console.time('libxml')
-		var doc = Libxml.parseXmlString(data)
-		console.timeEnd('libxml')
-		var ToString = doc.toString
-		if (ToString.name != 'xxxxx') {
-			doc.toString = function xxxxx() {
-				return ToString.apply(this, arguments).replace(/^\s+|\s+$/g, '')
-			}
-		}
-		//console.error(doc.toString())
-		return doc
-	} else {
-		console.warn('libxml is not installed')
-	}
-}
-
-function domjs(data) {
-	console.time('dom-js')
-	var doc
-	new DomJS().parse(data, function (err, dom) {
-		doc = dom
-	})
-	console.timeEnd('dom-js')
-
-	doc.toString = function () {
-		return doc.toXml()
-	}
-	return doc
-}
-var maxRandomAttr = parseInt(Math.random() * 60)
-console.log('maxRandomAttr', maxRandomAttr)
-function addAttributes(el) {
-	var c = parseInt(Math.random() * maxRandomAttr)
-	while (c--) {
-		el.setAttribute('dynamic-attr' + c, c + new Array(c).join('.'))
-	}
-	var child = el.firstChild
-	while (child) {
-		if (child.nodeType == 1) {
-			addAttributes(child)
-		} else if (child.nodeType == 4) {
-			//cdata
-			el.insertBefore(el.ownerDocument.createTextNode(child.data), child)
-			el.removeChild(child)
-		}
-		child = child.nextSibling
-	}
-}
-
-// Create a Test Suite
-describe('XML Node Parse', () => {
-	it('big file parse', () => {
```

Comment:
Why remove this whole test?

Reply:
Here are my reasons (should have highlighted them in advance): 
It was testing that the "performance" (measured in execution time) is better then two other libraries that are not listed in our devDependencies and that are also no longer actively maintained. It was also one of the slowest tests overall.

If I remember correctly, it was doing the comparison by modifying the prototype of DOMParser, so it modified global state.

Reply:
I would favor using `xit` or `xdescribe` instead, and maybe do this in a separate PR.